ir/node.cpp

```diff
@@ -102,6 +102,17 @@ Util::JsonObject* IR::Node::sourceInfoJsonObj() const {
     return json;
 }
 
+static cstring quote(cstring s) {
```

Comment:
This quoting is really specific to JSON.
For C you would need more cases.
So I decided to make it local for now.